### PR TITLE
Phase Diagram App updated docs

### PR DESCRIPTION
**Pull request by mattmcdermott**
@tschaume ready to merge

### CODE REVIEW

**Comment by mattmcdermott** [comment]
#13

File: docs/user-guide/phase-diagram.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+## Manual
+
+### Introduction 
+Phase diagrams represent the thermodynamic phase equilibria of multicomponent systems and reveal useful insights into fundamental material aspects regarding the processing and reactions of materials. However, the experimental determination of a phase diagram is an extremely time-consuming process, requiring careful synthesis and characterization of all phases in a chemical system.
+
+Computational tools can accelerate phase diagram construction significantly. By calculating the energies of all known compounds in a given chemical system (e.g., Li-Fe-O), we can determine the phase diagram for that system at 0 K and 0 atm. Furthermore, for systems comprised of predominantly solid phases open with respect to a gaseous element, approximations can be made as to the finite temperature and pressure phase diagrams.
+
+In this manual, we will outline the general usage and thermodynamics methodology of our computational phase diagrams app. The details of the calculations, including computational methodology and accuracy, used in constructing these phase diagrams are available in the *Calculations Manual* and will not be further covered in this manual.
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: link to the calculations manual?

File: docs/user-guide/phase-diagram.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+### Using the Computational Phase Diagram App
+To construct the phase diagram of a 2-4 component chemical system (e.g., Li-Fe-O), you only need to select the desired elements from the periodic table and click the "Generate" button.
+
+To generate a grand potential phase diagram, for which one of the components of the phase diagram is considered 'open' to the system, select a projected element from the drop-down list, and input a desired chemical potential either manually or using the temperature slider (for supported elements, $\ce{O2}$, $\ce{N2}, $\ce{H2}, $\ce{F2}, $\ce{Cl2} only). When available, the temperature slider has been calibrated using entropy data from NIST. [^1]
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: H2 and Cl2 don't render correctly

File: docs/user-guide/phase-diagram.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+* Checking the 'Show Unstable' box will display the compositions at which unstable compounds have been computed as blue circles.
+* You can zoom into a portion of the phase diagram by defining a zoom area: click and hold at the upper-left corner of the desired area, and drag down and right, releasing the mouse at the lower-right corner of the desired area. A pop-up with the zoomed-in portion should appear. To return to the original phase diagram, click anywhere outside the pop-up.
+
+### Interpreting Phase Diagrams
+This section provides a short guide on interpreting phase diagrams generated by the PDApp. A more comprehensive discussion and example of phase diagram interpretation can be found in refs [^2] and [^3].
+#### Basic Phase Diagram Information
+While the PDApp generates several types of phase diagrams, some information is common to all phase diagrams. As an illustrative example, we refer to *Figure 1*, a ternary Li-Fe-O phase diagram at 0 K and 0 atm calculated using the PDApp.
+
+![Calculated Li-Fe-O Phase Diagram](docs/img/phase-diagram/Li-Fe-O_calculated.png "Calculated Li-Fe-O Phase Diagram")
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: image doesn't load

File: docs/user-guide/phase-diagram.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+The PDApp models only the thermodynamics of a system. Thus, compounds that are metastable (kinetically stabilized) may not show up as stable on the generated phase diagram. We expect that compounds for which the decomposition energy into stable phases is low are potential candidates for metastability; however, compounds with extremely high decomposition energies into stable phases (i.e., over about 200 meV/atom) are less likely to be metastable.
+
+#### Compositional Phase Diagrams
+In a compositional phase diagram, the system is closed with respect to the environment. We display compositional phase diagrams involving different numbers of components in slightly different ways.
+
+##### Binary Compositional Phase Diagrams
+*Figure 2* is an example of a calculated binary Fe-P phase diagram at 0 K and 0 atm using the PDApp. Binary phase diagrams show the complete convex hull for the system, where the y-axis is the formation energy per atom and the x-axis is the composition (e.g., in the Fe-P phase diagram, the x-axis is the fraction of P).
+
+![Calculated Fe-P Phase Diagram](docs/img/phase-diagram/Fe-P_calculated.png "Calculated Fe-P Phase Diagram")
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: image doesn't load

File: docs/user-guide/phase-diagram.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+The lines in quaternary phase diagrams define polyhedra with 4 vertices each rather than triangles. At any point in the phase diagram other than the stable nodes, the equilibrium phases are given by the vertices of the polyhedron bounding that composition.
+
+#### Grand Potential Phase Diagrams
+Grand potential phase diagrams are phase diagrams representing phase equilibria that are open to a particular element. In these phase diagrams, the chemical potential of an element is an external input variable that the user must specify. In some cases (e.g., a gaseous element such as oxygen), the chemical potential can be related to temperature and partial pressure via thermodynamic relations and known experimental data. [^5]
+
+A grand potential phase diagram involving n components is presented in the same way as a compositional phase diagram involving $n-1$ components.
+
+![Calculated Li-Fe-O Grand Potential Phase Diagram](docs/img/phase-diagram/Li-Fe-O_700K.png "Calculated Li-Fe-O Grand Potential Phase Diagram")
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: image doesn't load

File: docs/user-guide/phase-diagram.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+*Figure 3* shows the calculated grand potential phase diagram for the Li-Fe-O system at an oxygen chemical potential consistent with 700 K and 0 atm. The phase diagram provides information on the equilibrium phases under these conditions for any Li:Fe composition (note that the O fraction is controlled by interactions with the environment). For example, we may observe that a compositon with a relatively high Fe ratio (near composition = 1) will comprise $\ce{Fe2O3} and $\ce{LiFeO2}. In addition, by examining the changes in the equilibrium phases as the chemical potential changes, we can obtain insights into how the phase equilibria are modified with changes in the chemical potential. A detailed discussion of this application can be found in references 2 and 3. [^2] [^3]
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: invalid math mode `$\ce{Fe2O3} and $\ce{LiFeO2}`

File: docs/user-guide/phase-diagram.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+### Accuracy of Calculated Phase Diagrams
+
+![Experimental Li-Fe-O Phase Diagram](docs/img/phase-diagram/Li-Fe-O_exp.png "Experimental Li-Fe-O Phase Diagram")
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: image doesn't load

File: docs/user-guide/phase-diagram.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+*Figure 4* shows the experimentally determined phase diagram for the Li-Fe-O system at 673K.[^4] Even though the experimental phase diagram is at a much higher temperature, we can see that our calculated phase diagram reproduces the features very well. The phases $\ce{Li2O}, $\ce{Fe3O4}, $\ce{Fe2O3}, $\ce{LiFeO2}, $\ce{Li5FeO4} are stable in both the experimental diagram and our calculated phase diagram. The only additional phase in our calculated diagram is $\ce{FeO}$, which is well known to be difficult to obtain in stoichiometric proportion under normal conditions. Our calculations correctly reproduce the $\ce{FeO} formation enthalpy to within experimental accuracy, and our calculated phase diagram is fullly consistent with known experimental thermochemical data along the Fe-O line at 1 atm and 298K.
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: `LiFeO2` doesn't render correctly.  
`$\ce{Li5FeO4}` end of math mode missing

File: docs/user-guide/phase-diagram.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+For grand potential phase diagrams, further approximations are made as to the entropic contributions.[^2][^3] They are therefore expected to be less accurate, but nonetheless provide useful insights on general trends.
+
+### Thermodynamics Methodology
+In the remainder of this section, we will use an Li-Fe-O system to illustrate the methodology without loss of generality.
+
+#### Compositional Phase Diagrams
+To construct a phase diagram, one would need to compare the relative thermodynamic stability of phases belonging to the system using an appropriate free energy model. For an isothermal, isobaric, closed system, the relevant thermodynamic potential is the Gibbs free energy, $G$, which can be expressed as a Legendre transform of the enthalpy, $H$, and internal energy, $E$, as follows:
+
+$$\eqalign{G(T,P,N_{Li},N_{Fe},N_{O}) &= H(T,P,N_{Li},N_{Fe},N_{O}) - TS(T,P,N_{Li},N_{Fe},N_{O}) &= E(T,P,N_{Li},N_{Fe},N_{O}) + PV(T,P,N_{Li},N_{Fe},N_{O})- TS(T,P,N_{Li},N_{Fe},N_{O})}$$
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: equation too long (break after second `=`)

File: docs/user-guide/phase-diagram.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+
+where $T$ is the temperature of the system, $S$ is the entropy of the system, $P$ is the pressure of the system, $V$ is the volume of the system, and $N_i$ is the number of atoms of species $i$ in the system.
+
+For systems comprising primarily of condensed phases, the $PV$ term can be neglected and at 0K, the expression for $G$ simplifies to just $E$. Normalizing $E$ with respect to the total number of particles in the system, we obtain $\bar{E}(0,P,x_{Li},x_{Fe},x_O), where $x_i = \frac{N_i}{N_{Li} + N_{Fe} + N_{O}}$. By taking the convex hull[^5] of $E$ for all phases belonging to the M-component system and projecting the stable nodes into the $(M - 1)$ -dimension composition space, one can obtain the 0 K phase diagram for the closed system at constant pressure. The convex hull of a set of points is the smallest convex set containing the points. For instance, to construct a 0 K, closed Li-Fe-O system phase diagram, the convex hull is taken on the set of points in $(\bar{E},x_{Li},x_{Fe})$ space with $x_O$ being related to the other composition variables by $x_O = 1 - x_{Li} - x_{Fe}$
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: end of math mode missing

File: docs/user-guide/phase-diagram.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+
+#### Grand Potential Phase Diagrams
+In many scientific applications, the phase equilibria of interest is not that of a closed system, but rather one which is open to a particular element. For instance, synthesis might be carried out in flowing Argon gas at a particular temperature, which would mean the system is open to gaseous elements such as oxygen. Other experiments may be carried out in air, which essentially serves as an infinite reservoir of atmospheric elements such as oxygen, nitrogen and others.
+
+In environments that are open to a particular element (say oxygen), the relevant control variable is the chemical potential of that element $\mu_{O}$. The relevant thermodynamic potential to study phase equilibria in an open system is the grand potential, which is defined as the following for an Li-Fe-O system that is open with respect to oxygen:
+
+$$\eqalign{\phi(T,P,N_{Li},N_{Fe},\mu_{O}) &= G(T,P,N_{Li},N_{Fe},\mu_{O}) - \mu_ON_O(T,P,N_{Li},N_{Fe},\mu_{O}) &= E(T,P,N_{Li},N_{Fe},\mu_{O}) - TS(T,P,N_{Li},N_{Fe},N_{O}) + \mu_ON_O(T,P,N_{Li},N_{Fe},\mu_{O})}$$
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: equation too long (break at second `=`)

File: docs/user-guide/phase-diagram.md [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+
+$$\bar{\phi} = \frac{E-\mu_ON_O}{N_{Li} + N_{Fe}}$$
+
+Using the above assumption, the effect of temperature and partial pressure can be fully captured in a single chemical potential variable, with a more negative value corresponding to higher temperatures or lower partial pressures. The chemical potential is then treated as an external variable to obtain a grand potential phase diagram under a particular condition by taking the convex hull of $\bar{\phi}$ for all phases and projecting the stable nodes into the Li-Fe composition space.
+
+### Citation
+To cite the Computational Phase Diagram App, please reference the following works:
```
Comment (tschaume) [inline docs/user-guide/phase-diagram.md]: make this a list

**Comment by mattmcdermott** [comment]
@tschaume @dwinston should be ready to merge now!

**Pull request merged**